src/workflow/restore.c

```diff
@@ -14,11 +14,11 @@
 
 #include <workflow/restore.h>
 
-#include <backup.h>
+#include <backup/backup.h>
```

Comment:
I don't believe restore.c actually needs backup.h. Didn't find any reference.
I know you just moved it around but this is a refactoring PR might as well remove.

Reply:
:+1: